src/main/java/com/gmail/val59000mc/threads/ElapsedTimeThread.java

```diff
@@ -50,30 +62,51 @@ public void run() {
 		if(time%intervalTimeEvent == 0){
 			
 			if(enableTimeEvent){
-				
-				String message = Lang.EVENT_TIME_REWARD
+				String message;
+
+				message = Lang.EVENT_TIME_REWARD
 						.replace("%time%", TimeUtils.getFormattedTime(intervalTimeEvent))
 						.replace("%totaltime%", TimeUtils.getFormattedTime(time))
-						.replace("%money%", ""+reward);
-				
-				for(UhcPlayer uhcP : playingPlayers){
-					
+						.replace("%money%", "" + reward);
+
+				for (UhcPlayer uhcP : playingPlayers) {
 					try {
 						Player p = uhcP.getPlayer();
-						VaultManager.addMoney(p, reward);
-						if(!message.isEmpty()){
+						if (reward > 0) {
+							VaultManager.addMoney(p, reward);
+						}
+						if (cmdsNeedsPlayer) {
+							timeCommands.forEach(cmd -> {
+								try {
+									Bukkit.dispatchCommand(Bukkit.getConsoleSender(), cmd.replace("%name%", uhcP.getRealName()));
+								} catch (CommandException exception) {
+									Bukkit.getLogger().warning("The command: '" + cmd + "' does not exists.");
+								}
+							});
+						}
+
+						if (!message.isEmpty()) {
 							p.sendMessage(message);
 						}
```

Comment:
Why do you send the message regardless of money received here? That's different on the other events.